src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,20 @@
+@mixin circle() {
+  height: 80vmin;
+  width: 80vmin;
+  border-radius: 50%;
+}
+
+@mixin center-size() {
+  height: 40px;
+  width: 40px;
+}
+
+@mixin second-hand-size() {
+  width: 10px;
+  height: 235px;
+}
+
+@mixin minute-hand-size {
```

Comment:
Missing parentheses for the minute-hand-size mixin declaration. Add '()' after the mixin name.